src/Facebook/InstantArticles/Client/Client.php

```diff
@@ -124,7 +124,7 @@ public function removeArticle($canonicalURL)
             $this->facebook->delete($articleID);
             return InstantArticleStatus::success();
         }
-        return InstantArticleStatus::notFound(['An Instant Article ID ' . $articleID . ' was not found for ' . $canonicalURL . ' in ' . __FUNCTION__ . '.']);
+        return InstantArticleStatus::notFound([ServerMessage::info('An Instant Article ID ' . $articleID . ' was not found for ' . $canonicalURL . ' in ' . __FUNCTION__ . '.')]);
```

Comment:
@scottfalkingham 

How about 

* adding and committing a failing test
* committing a fix

?